packages/sdk/src/HopBridge.ts

```diff
@@ -575,10 +594,17 @@ class HopBridge extends Base {
     sourceChain = this.toChainModel(sourceChain)
     destinationChain = this.toChainModel(destinationChain)
 
-    const requireFee =
-      destinationChain?.isL1 ||
-      (!sourceChain?.equals(Chain.Ethereum) &&
-        destinationChain?.equals(Chain.Optimism))
+    let requireFee = false
+    if (destinationChain?.isL1) {
```

Comment:
The tx fees on polygon and xDai are negligible (< $0.01 and lower). Because of this, including those fees in the calcs would effectively make no difference, especially given that there is already (a) amm slippage and (b) a bonder fee.

Does it make sense to remove this conditional completely in order to simplify and charge a tx fee on every network?

Reply:
Sounds good; it's removed so all chains have same fee requirement here